pyproject.toml

```diff
@@ -5,22 +5,16 @@ description = ""
 authors = ["denys <denysburimov@gmail.com>"]
 
 [tool.poetry.dependencies]
-python = "^3.9"
-Flask = "^2.1.0"
-Flask-SQLAlchemy = "^2.5.1"
-Flask-Login = "^0.5.0"
-click = "^8.1.0"
-email-validator = "^1.1.3"
-Flask-WTF = "^1.0.0"
-WTForms = "^3.0.1"
-itsdangerous = "^2.1.2"
-Jinja2 = "^3.1.1"
-MarkupSafe = "^2.1.1"
-python-dotenv = "^0.20.0"
-SQLAlchemy = "^1.4.32"
-Werkzeug = "2.0.0"
-psycopg2-binary = "^2.9.3"
-Flask-Migrate = "^3.1.0"
+python = "^3.11"
```

Comment:
I am a supporter of strict versions in pyproject.toml

Reply:
^3.11 means also 3.11.2, 3.11.3 etc